coder_sniffer/Drupal/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -95,8 +95,17 @@ public function process(File $phpcsFile, $stackPtr)
                 continue;
             }
 
+            // If there is a phpstan-ignore inline comment disregard it and continue searching backwards
+            // to find the function comment.
+            if ($tokens[$commentEnd]['code'] === T_COMMENT
+                && preg_match('/\@phpstan-ignore/', $tokens[$commentEnd]['content']) === 1
```

Comment:
strpos is faster than regex, please use that.

Reply:
ok will do.